src/sentry/api/endpoints/chunk.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, file):
         super().__init__(data)
 
 
-@pending_silo_endpoint
+@all_silo_endpoint
```

Comment:
```suggestion
@region_silo_endpoint
```

Looking at the URLs for this endpoint, it only has a single URL set and the path for that URL includes both an organziation and project slug, so we can easily route requests to this endpoint to the correct regions.